### PR TITLE
Cthun Tech tree

### PR DESCRIPTION
**Pull request by SebastianTanoni**

### CODE REVIEW

File: src/WarcraftLegacies.Source/Factions/Ahnqiraj.cs
Comment (YakaryBovine) [inline src/WarcraftLegacies.Source/Factions/Ahnqiraj.cs:52]: Let's get these using proper `Constants`. It's a new file so no point having tech debt straight away!

Comment (SebastianTanoni) [reply src/WarcraftLegacies.Source/Factions/Ahnqiraj.cs:52]: image
These are all the boat stuff, since we were discussing cthun not having boats, i decided not to touch it for now!

Comment (SebastianTanoni) [reply src/WarcraftLegacies.Source/Factions/Ahnqiraj.cs:52]: i can just remove it

Comment (YakaryBovine) [reply src/WarcraftLegacies.Source/Factions/Ahnqiraj.cs:52]: Makes sense. Nothing gets reviewed going from `dev` to `main`, so everything going into `dev` _either_ needs to be technically correct or have a ticket to finish it off.

So if this stuff is just a placeholder and you want to keep it in for now, you just need to create another Issue to clean it up (and as you say, maybe I end up picking up that Issue and doing some custom mechanic instead of the boats).

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
As discussed I'll be fairly limited in my review this time. I'm just going to focus on:
- Bugs
- Any instance of a problem where I'm not certain of the solution you want
- Icons

I won't worry about grammar and tooltip accuracy overmuch, as it will be faster if I just cover that myself.

**Comment by YakaryBovine** [comment]
image
Doesn't have Unsummon Building.

**Comment by YakaryBovine** [comment]
Pyramid missing DISBTN icon: 
image

**Comment by YakaryBovine** [comment]
Some random dashes in tooltips. These are the ones I found but have a look for the remainder: 
image
image

**Comment by YakaryBovine** [comment]
image
Can't actually research Navigation.

**Comment by YakaryBovine** [comment]
About half of the buildings say "Summon" and the other half say "Build", e.g. this one. They should all be Summon as they're using the Undead building type.

**Comment by YakaryBovine** [comment]
Unit names here aren't accurate, the first two should say Illithid in front:
image

**Comment by YakaryBovine** [comment]
Web research refers to Crypt Friends which AQ doesn't have: 
image

**Comment by YakaryBovine** [comment]
Random unlabelled ability here, presumably this should be hidden: 
image

**Comment by YakaryBovine** [comment]
Improved Swarm Beetle tooltip says it upgrades to 0:
image

**Comment by YakaryBovine** [comment]
This trains Silithid Tunneler, not Tunneler: 
image

**Comment by YakaryBovine** [comment]
Should be Dispel Magic, not Dispel: 
image

**Comment by YakaryBovine** [comment]
Earth Shackles shouldn't share an icon with Cripple. 
image

And the buff shouldn't share an icon with Aerial Shackles, nor have a different icon to the spell.
image

That said, I think there are a number of Shackles in the game with different names that probably need to be reconciled, so if you want me to just solve this in #2845 alongside those I can do that.

**Comment by YakaryBovine** [comment]
Same thing here, unit names don't correspond to the unit names it actually trains:
image

**Comment by YakaryBovine** [comment]
Should say Research in front.
image

**Comment by YakaryBovine** [comment]
Stone Armor buff actually says Mage Armor and has the wrong icon
image

**Comment by YakaryBovine** [comment]
image
Doesn't actually train anything. Is it just a placeholder? If so just create an Issue for finishing it and respond to this comment with it.

**Comment by YakaryBovine** [comment]
Every building tooltip should be updated to indicate it has this Ahn'Qiraj Spirits ability:
image

**Comment by YakaryBovine** [comment]
Plague Strike has a different Normal tooltip to its Learn tooltip:
image

**Comment by YakaryBovine** [comment]
This isn't how Normal tooltips are formatted, shouldn't have the Level 1 thing in the description: 
image

**Comment by YakaryBovine** [comment]
"Seconds seconds": 
image

**Comment by YakaryBovine** [comment]
Roar of Despair shouldn't share an icon with Howl of Terror.
Also the buff should be negative, not positive - did you base it off the wrong ability? Should be based off Howl of Terror.
image

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
Ok all done!
The actual content is fucking awesome BTW, very C'thun vibes with some cool abilities. Just needs a lot of polish. As mentioned I'll do the complex stuff in #2845, but there are also a lot of simple mistakes to rectify for you.

**Comment by YakaryBovine** [comment]
image
Icon still an issue

**Comment by YakaryBovine** [comment]
Random dash here 
image

**Comment by YakaryBovine** [comment]
Web tooltip still mentions Crypt Fiends: 
image

File: mapdata/WarcraftLegacies/AbilityData/Skin/1195847745.json
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/1195847745.json:21]: As discussed, let's get this Spell Shield using the new button position trick instead of the old Spell Book trick.

**Comment by YakaryBovine** [comment]
Still need to remove the full stop at the end of the ability list: 
image

**Comment by YakaryBovine** [comment]
Tooltip indicates it attacks trees, but trees don't exist: 
image

**Comment by YakaryBovine** [comment]
This one still says "seconds seconds" on the level 1 tooltip: 
image

**Comment by YakaryBovine** [comment]
Something I didn't notice before, this Stone Armor ability violates some rules.
The effect is identical to Dalaran's Mage Armor, so it should be called Mage Armor. We doubly can't call it Stone Armor because Dalaran's Earth Golems already have that.
image

**Comment by YakaryBovine** [comment]
Good name change, but remove the word "holy"! I'd just say "Binds the target in shackles, immobilising them..." etc
image

**Comment by YakaryBovine** [comment]
Another one I just noticed... We can't use this button position because it's occupied by the Cancel button: 
I suggest moving it to the top right since Ahn'qiraj buildings don't have an Attack and this is basically their Attack.
image

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
Good changes but still a fair bit to fix. Found some new issues but many of these are just leftover from the old set.

**Comment by SebastianTanoni** [comment]
> Another one I just noticed... We can't use this button position because it's occupied by the Cancel button: I suggest moving it to the top right since Ahn'qiraj buildings don't have an Attack and this is basically their Attack. image

1 of cthun building has an attack. Would you still put it there?

Or I could remove the attack from the t3

**Comment by SebastianTanoni** [comment]
> image Icon still an issue

I don't understand what's wrong here... il have another look

**Comment by SebastianTanoni** [comment]
Okay, The pyramid icon herrdave is on the case.

The web thing im waiting for your response.

And the spell shield im waiting for your response, the fix isnt working

**Review by YakaryBovine — APPROVED** [review]
Nice, looking good!
I'm approving now but I can't merge it in due to a merge issue I'm confused about. I'll ping you on Discord about it.

**Review by YakaryBovine — APPROVED** [review]
False alarm, resolved the conflict

**Pull request merged**
